cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -102,6 +102,23 @@ else()
   hunter_fatal_error("TODO: set toolset for boost" WIKI "error.boost.toolset")
 endif()
 
+if("@ANDROID@")
+  set(toolset_version ndk)
+endif()
+
+get_directory_property(defs COMPILE_DEFINITIONS)
+foreach(def ${defs})
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D${def}")
```

Comment:
Also this code will be used while building MSVC compiler. So either we need to put it under `if(NOT MSCV)` or may be `if(ANDROID)`.